src/core.c/Array.pm6

```diff
@@ -1322,7 +1322,11 @@ my class Array { # declared in BOOTSTRAP
         nqp::isnull($!descriptor) ?? Nil !! $!descriptor.name
     }
     method of() {
-        nqp::isnull($!descriptor) ?? Mu !! $!descriptor.of
+        nqp::isconcrete(self)
```

Comment:
Why not make a multi out of it and benefit from dispatch caching?

Reply:
NO real reason, I wanted to keep the diff as small as possible, to at least gauge whether this would be considered the right approach.  Because making it a multi then poses the question whether it should become available for every type, (aka having a Mu.of fallback) or not.

Also note that this would need to be done for `.returns` and `.name` as well.